kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -98,10 +100,9 @@ object KyuubiApplicationManager {
   }
 
   private def setupFlinkK8sTag(tag: String, conf: KyuubiConf): Unit = {
-    // TODO: yarn.tags or flink.yarn.tags, the mess of flink settings confuses me now.
```

Comment:
Check the Flink configuration documentation, there is no such configuration as `flink.yarn.tags`.

Although Flink supports converting `flink.yarn.xx` to `yarn.xx`, there is no `yarn.tags` configuration in yarn.